graphql-protocol/src/main/java/com/linecorp/armeria/server/graphql/protocol/AbstractGraphqlService.java

```diff
@@ -150,6 +191,18 @@ protected HttpResponse doPost(ServiceRequestContext ctx, HttpRequest request) th
         return unsupportedMediaType();
     }
 
+    private void bindMultipartVariable(Map<String, Object> variables, List<String> value,
+                                       List<MultipartFile> multipartFiles) {
```

Comment:
It seems like `multipartFiles` is `@Nullable`

```suggestion
                                       @Nullable List<MultipartFile> multipartFiles) {
```

Reply:
Doesn't FileAggregatedMultipart get even an empty list? 🤔

Reply:
I see 😅  Is it guaranteed that the `key` always exists?

https://github.com/line/armeria/blob/5119a71881f3428057205f8a877a809045d02269/graphql-protocol/src/main/java/com/linecorp/armeria/server/graphql/protocol/AbstractGraphqlService.java#L122

Reply:
Oh.. I didn't understand it well.

I fixed